### PR TITLE
React and Redux Group Project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by MozamelJawad**
In this **PR**, we created the real live data from the SpaceX API. We build a web application for a company that provides commercial and scientific space travel services. The application allows users to book rockets and join selected space missions.
The project has the following parts:

- **Rockets** 
The Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation button or cancel the previously made booking. 

**- Missions**
The Missions section displays a list of current missions along with their brief description and participation status. There is also a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier.

**- My Profile**
- The My Profile section displays all reserved rockets and space missions.**

### CODE REVIEW

File: src/redux/missions/missions.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+.mission-table td,
+.mission-table th,
+.mission-table {
+  border: 1px solid #ddd;
+  padding: 8px;
+}
+
+.mission-table th {
+  padding-top: 12px;
```
Comment (ghost) [inline src/redux/missions/missions.css]: - [ OPTIONAL] 👋 Hey there! Great job on your PR! 🎉 Just one friendly suggestion: let's keep our project organized. Please avoid putting CSS files in the Redux folder. Instead, consider creating a separate "styles" or "css" folder for your stylesheets. This helps keep our codebase clean and makes it easier for everyone to find what they need. Thanks for your hard work, and keep up the good coding! 😊🚀 #KeepItTidy

File: src/__test__/__snapshots__/Missions.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <tr>
+          <th>
+            Mission
+          </th>
+          <th>
+            Description
+          </th>
+          <th
+            class="status"
+          >
```
Comment (ghost) [inline src/__test__/__snapshots__/Missions.test.js.snap]: - [ ] 👋 Hey team! You're doing a great job with your React testing library for component rendering. 🧪🚀 However, for a more comprehensive test coverage, it'd be fantastic if we could also include unit tests for our pure functions, especially in the Redux part of our codebase. This ensures that our logic is solid and dependable. 🧐 Let's aim for that well-rounded test suite! Thanks for your dedication to quality code.

**Review by ghost — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights
✅ The web app is functioning properly
✅ Followed Design Guidelines 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @MozamelJawad and @mershark👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented changes requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**